src/FixedPairList.cpp

```diff
@@ -274,6 +282,7 @@ namespace espressopp {
       .def("add", pyAdd)
       .def("size", &FixedPairList::size)
       .def("getBonds",  &FixedPairList::getBonds)
+      .def("remove",  &FixedPairList::remove)
```

Comment:
@niktre, why would renaming <remove> to <removeAll> fix anything? It is not removing all fixedXLists, It seems to be misleading.... However I fixed the tabs @junghans asked for..